screens/Home.js

```diff
@@ -127,25 +110,17 @@ const HomeScreen = () => {
       </AnimateChildrenIn>
 
       {cardData.length ? (
-        <FlatList
```

Comment:
i was tired of those warnings about a `FlatList` inside of a `ScrollView` 😅 
also, it wasn't really necessary..
also, if you have a `FlatList` inside of a `ScrollView`, we don't get the benefits of the `VirtualizedList`, which recycles cells, so that's super important for the Groups tab